lib/workers/repository/config-migration/branch/index.ts

```diff
@@ -18,10 +22,50 @@ export async function checkConfigMigrationBranch(
     return null;
   }
   const configMigrationBranch = getMigrationBranchName(config);
-  if (await migrationPrExists(configMigrationBranch)) {
+  const commitMessageFactory = new ConfigMigrationCommitMessageFactory(
+    config,
+    migratedConfigData.filename
+  );
+  const prTitle = commitMessageFactory.getPrTitle();
+  const closedPrConfig: FindPRConfig = {
+    branchName: configMigrationBranch,
+    prTitle,
+    state: PrState.Closed,
+  };
+
+  const branchPr = await migrationPrExists(configMigrationBranch); // handles open/autoClosed PRs
+
+  if (!branchPr) {
+    // handles closed PR
+    const closedPr = await platform.findPr(closedPrConfig);
+
+    // found closed migration PR
+    if (closedPr) {
+      logger.debug(
+        { prTitle: closedPr.title },
+        'Closed PR already exists. Skipping branch.'
+      );
+      const branchConfig: BranchConfig = {
+        prTitle,
+        manager: '',
+        upgrades: [],
+        branchName: configMigrationBranch,
+        userStrings: {
+          ignoreTopic: 'Renovate Ignore Notification',
+          ignoreMajor: '',
+          ignoreDigest: '',
+          ignoreOther: '',
+        },
+        suppressNotifications: config.suppressNotifications,
+      };
+      await handlepr(branchConfig, closedPr);
```

Comment:
I think this is too hacky. What do you think about skipping use of `handlepr` and writing an equivalent function specific for this use case?

Reply:
added standalone function for it